README.md

```diff
@@ -2,36 +2,43 @@
 
 ## Overview
 
-A brief description. Be clear about the purpose of the resolver,
-its capabilities and limitations.
+A custom Jinja filter for Sceptre for unquoting resolvers appearing in var files.
 
 ## Installation
 
 Installation instructions
 
 To install directly from PyPI
 ```shell
-pip install sceptre-resolver-template
+pip install jinja-unquote-resolvers-filter
 ```
 
 To install from the git repo
 ```shell
-pip install git+https://github.com/Sceptre/sceptre-resolver-template.git
+pip install git+https://github.com/Sceptre/jinja-unquote-resolvers-filter.git
 ```
 
 ## Usage/Examples
 
-Refer to the [Sceptre resolver docs](https://docs.sceptre-project.org/latest/docs/resolvers.html#resolvers)
-for more information.
+In your var file:
 
 ```yaml
-parameter|sceptre_user_data:
-  <name>: !custom <args>
+Subnets:
+  - '!stack_output_external mystack::subnet_a'  # Quotes needed to protect a YAML tag.
```

Comment:
This comment is helpful, but I think it would make sense to explicitly describe how this works and why you need to put the yaml tag into quotes.

Reply:
hopefully the motivation section I just added will make this clearer.

Reply:
What about multi-line resolver arguments, such as when a resolver takes a dict or a list argument?

Reply:
Also, what about resolvers that take resolvers as argument?

Such as in the case like the `!sub` resolver: https://docs.sceptre-project.org/4.1.0/docs/resolvers.html#sub